modules/core/shared/src/main/scala/io/circe/Encoder.scala

```diff
@@ -23,40 +23,39 @@ import cats.data.Chain
 import cats.data.NonEmptyChain
 import cats.data.NonEmptyList
 import cats.data.NonEmptyMap
-import cats.data.NonEmptySet
 import cats.data.NonEmptySeq
+import cats.data.NonEmptySet
 import cats.data.NonEmptyVector
 import cats.data.OneAnd
 import cats.data.Validated
 import io.circe.`export`.Exported
 
 import java.io.Serializable
 import java.net.URI
-import java.time.{
-  Duration,
-  Instant,
-  LocalDate,
-  LocalDateTime,
-  LocalTime,
-  MonthDay,
-  OffsetDateTime,
-  OffsetTime,
-  Period,
-  Year,
-  YearMonth,
-  ZoneId,
-  ZoneOffset,
-  ZonedDateTime
-}
-import java.time.format.{ DateTimeFormatter, DateTimeFormatterBuilder, SignStyle }
-import java.time.format.DateTimeFormatter.{
-  ISO_LOCAL_DATE_TIME,
-  ISO_LOCAL_TIME,
-  ISO_OFFSET_DATE_TIME,
-  ISO_OFFSET_TIME,
-  ISO_ZONED_DATE_TIME
-}
-import java.time.temporal.{ ChronoField, TemporalAccessor }
+import java.time.Duration
+import java.time.Instant
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.time.LocalTime
+import java.time.MonthDay
+import java.time.OffsetDateTime
+import java.time.OffsetTime
+import java.time.Period
+import java.time.Year
+import java.time.YearMonth
+import java.time.ZoneId
+import java.time.ZoneOffset
+import java.time.ZonedDateTime
+import java.time.format.DateTimeFormatter
+import java.time.format.DateTimeFormatter.ISO_LOCAL_DATE_TIME
```

Comment:
personally, I don't like these import reformats. I think grouped imports make it easier to see the package level dependencies and I would rather use `{ }` to the maximum vs the minimum.

Reply:
I don't have strong opinions on it, other than "let the tooling handle it" 😅 

In this case, it was the result of running `scalafixAll` and `scalafmtAll` (then reverting the changes to the files I hadn't touched).

Reply:
I can revert the changes manually if you find it particularly objectionable, but it might be worth revisiting the formatting config if that is the case 🤷🏻

Reply:
Also, in my experience, unfolded imports tend to trigger less conflicts on merges and rebases. I guess this could one of the reasons why it was made a default behavior for the OrganizeImports rule in scalafix.

Reply:
I needed to re-run the CI, so I've rolled back the import reformat